src/electron/views/automated-checks/automated-checks-view.tsx

```diff
@@ -216,6 +219,34 @@ export class AutomatedChecksView extends React.Component<AutomatedChecksViewProp
         );
     }
 
+    private renderReflowCommandBar(
```

Comment:
This method should be called on line 258 where we are showing the component that is enabled when the feature flag is enabled.